packages/header/README.md

```diff
@@ -1,10 +1,21 @@
 # Data Modelling Framework - UiPlugin - Header
 
-The `header` plugin is meant to be used as a web application header. Providing user information, application information, and an application recipe selector.
+The `header` plugin is meant to be used as a web application header. Providing user information, application
+information, and an application recipe selector. Below the header, another UI plugin is displayed.
 
 Add the package to your project with `npm add @development-framework/header`.
 
 If you want to specify a config object in some UiRecipe-entity, the Blueprint is included in the npm package.
 
 Copy it like so: `cp -R node_modules/@development-framework/header/dist/blueprint ./myApplicationBlueprints/`
 
+# Config
+
+The Header plugin can accept a config of type `HeaderPluginConfig` (see `blueprints/HeaderPluginConfig.json`).
+Explanation of the attributes:
+
+* `uiPluginsList`: Can be used to specify a list of UI plugins the user can select. By default, the fist UI plugin in
+  the list is displayed. If the list is empty, no plugin is displayed. NB! All plugins listed here must be included in
```

Comment:
Really think the header should use all recipes if the "uiPluginsList" is empty.
Sane defaults and minimum amount of config being the goal.